docs/api.md

```diff
@@ -191,10 +191,10 @@ GET /api/v1/accounts/:account_id/credentials/:credential_id
 
 **Path Parameters**
 
-| Name       | Description           |
```

Comment:
there was some inconsistency between item_id and credential_id so just standardized to credential_id